packages/validator-ajv8/test/usePrecompiledValidator.test.ts

```diff
@@ -0,0 +1,48 @@
+import { RJSFSchema } from '@rjsf/utils';
+
+import AJV8PrecompiledValidator from '../src/precompiledValidator';
+import { Localizer, ValidatorFunctions } from '../src';
+import * as superSchemaFns from './harness/superSchema';
+import superSchema from './harness/superSchema.json';
+import usePrecompiledValidator from '../src/usePrecompiledValidator';
+
+jest.mock('../src/precompiledValidator');
+
+type TestType = {
+  foo: string;
+  bar: boolean;
+};
+
+const validateFns = superSchemaFns as ValidatorFunctions;
+const rootSchema = superSchema as RJSFSchema;
+
+describe('usePrecompiledValidator()', () => {
+  describe('passing validatorFns and rootSchema to usePrecompiledValidator', () => {
+    let custom: any;
+    beforeAll(() => {
+      (AJV8PrecompiledValidator as unknown as jest.Mock).mockClear();
```

Comment:
You can probably use `jest.mocked()` instead of this sequence of casts - https://jestjs.io/docs/mock-function-api/#jestmockedsource-options

Reply:
Wow, I just learned something new

Reply:
I just learned about it recently 🙂